app/models/enterprise.rb

```diff
@@ -19,6 +19,7 @@ class Enterprise < ApplicationRecord
   preference :shopfront_producer_order, :string, default: ""
   preference :shopfront_order_cycle_order, :string, default: "orders_close_at"
   preference :shopfront_product_sorting_method, :string, default: "by_category"
+  preference :invoice_order_by_supplier, :boolean, default: false
```

Comment:
I changed my mind. Leave it as a preference. That's makes it easier to change the default at some point. Even though I don't like the preference system, it's a good choice here.